portfolio/src/main/webapp/index.html

```diff
@@ -1,175 +1,245 @@
-                <p>Throughout high school, I was involved in several extracurriculars- including teaching students using Google CS First, volunteering at the Liberty Science Center, and working with our school's chapter of the National Honor Society to help out with community events. I am glad that I was able to give back to my community when I had the chance. </p>
-                <p>After high school, I was unsure about whether I wanted to pursue a career in computer science or business. Computer science - I was interested in but scared to pursue. Business - I hated, but it was a safe option. Luckily, I was accepted into Google CSSI, where I met 29 other very accomplished students with similar goals and ambitions. In a short 3 weeks, we became unexpectedly close. To this day, we hang out and even attend hackathons together. </p>
-                <p>In college, I have become an active leader in the Honors Academy in the School of Engineering. I help interview prospective students and run panels to answer FAQ's. I love to get involved, and I'm glad Google SPS will be a step in the right direction. </p>
-             </div>
-
-             <div class="card">
-                <h2>What I enjoy to photograph</h2>
-                <p>I love to photograph anything that can trace back to a memory. Here are some pictures I have taken. </p>
-                <div class="row"> 
-
-                    <div class="column">
-                        <img src="/images/1.png" style="width:100%">
-                        <img src="/images/2.png" style="width:100%">
-                        <img src="/images/4.png" style="width:100%">
-                        <img src="/images/9.png" style="width:100%">
-                    </div>
-
-                    <div class="column">
-                        <img src="/images/3.png" style="width:100%">
-                        <img src="/images/8.png" style="width:100%">
-                    </div>  
-
-                    <div class="column">
-                        <img src="/images/5.png" style="width:100%">
-                        <img src="/images/6.png" style="width:100%">
-                        <img src="/images/7.png" style="width:100%">
-                    </div>
-
+        <div class="rightcolumn">
+          <div class="card">
+            <button class="button" onclick="addRandomGreeting()">
+              Click for a fun fact about me
+            </button>
+            <br />
+            <br />
+            <br />
+            <div id="greeting-container"></div>
+
+            <form action="/text" method="POST">
+              <p>Enter some words you like (seperated by commas)</p>
+              <textarea class="textarea" name="text-input">
+Ex. Broccoli, Reverie, Crisp, Whale, etc.</textarea
+              >
```

Comment:
I think the way you intended to use the examples here are better placed in the [`placeholder`](https://www.w3schools.com/tags/att_textarea_placeholder.asp) attribute:
```suggestion
              <textarea class="textarea" placeholder="Ex. Broccoli, Reverie, Crisp, Whale, etc." name="text-input" />
```